lib/index.js

```diff
@@ -139,7 +139,7 @@ const reducer = (inner, tag) => inner.replace(exps[tag], parse[tag]);
 const tags = [
   'include', 'includeContent', 'endInclude',
   'block', 'blockEnd', 'put', 'encode',
-  'comment', 'if', 'exists', 'elseIf',
+  'comment', 'if', 'classes', 'exists', 'elseIf',
```

Comment:
@joeosburn this should be deleted right?

Reply:
Yep